src/frontend/apps/impress/src/features/footer/Footer.tsx

```diff
@@ -16,9 +20,42 @@ const BlueStripe = styled.div`
 `;
 
 export const Footer = () => {
-  const { t } = useTranslation();
-  const { themeTokens } = useCunninghamTheme();
-  const logo = themeTokens.logo;
+  const { data: config } = useConfig();
+  const footerJson = config?.theme_customization?.footer;
+  const { i18n } = useTranslation();
+  const [legalLinks, setLegalLinks] = useState<Link[]>();
+  const [externalLinks, setExternalLinks] = useState<Link[]>();
+  const [bottomInformation, setBottomInformation] =
+    useState<BottomInformation>();
+  const [logo, setLogo] = useState<Logo>();
+  const resolvedLanguage = i18n.resolvedLanguage;
+
+  useEffect(() => {
+    if (!footerJson) {
+      return;
+    }
+
+    const langData = footerJson[resolvedLanguage as keyof typeof footerJson];
+    setLogo(langData?.logo || footerJson?.default?.logo);
+
+    setLegalLinks(langData?.legalLinks || footerJson?.default?.legalLinks);
+
+    const externalLinks =
+      langData && 'externalLinks' in langData
+        ? langData?.externalLinks
+        : footerJson?.default?.externalLinks;
+    setExternalLinks(externalLinks);
+
+    const bottomInformation =
+      langData && 'bottomInformation' in langData
+        ? langData?.bottomInformation
+        : footerJson?.default?.bottomInformation;
+    setBottomInformation(bottomInformation);
```

Comment:
Why not make an object of type "Content Type" directly instead of making 3/4 states